content-src/components/Spotlight/Spotlight.js

```diff
@@ -129,13 +146,24 @@ const Spotlight = React.createClass({
     return (<section className="spotlight">
       <h3 className="section-title">Highlights <Hint id="highlights_hint" title="Highlights" body={HIGHLIGHTS_HINT_TEXT} /></h3>
       <ul className="spotlight-list">
-        {sites.map((site, i) => <SpotlightItem
-          index={i}
-          key={site.guid || site.cache_key || i}
-          page={this.props.page}
-          source="FEATURED"
-          onClick={this.onClickFactory(i, site)}
-          {...site} />)}
+        {sites.map((site, i) => {
+          if (this.props.placeholder) {
```

Comment:
I'm curious how this works. If the data isn't ready, I expected `sites` could be empty. If so, we won't show any placeholders. Or is `sites` somehow hydrated with placeholders?